lib/ex_dtls.ex

```diff
@@ -243,6 +254,16 @@ defmodule ExDTLS do
   def handle_call(:get_cert, _from, %State{cnode: cnode} = state),
     do: {:reply, Unifex.CNode.call(cnode, :get_cert), state}
 
+  @impl true
+  def handle_info({:retransmit, message, counter}, state) do
+    if message != :handshake_want_read and counter < 60 and not state.finished? do
+      send(state.parent, {:retransmit, 1, message})
```

Comment:
instead of manually passing the parent pid, let's take it from the `from` argument of `handle_call`

Reply:
I'm not sure this will work, because the this message we will get by Process.send_after, so argument from in handle_call will be the process itself, if I good understand how handle_call works

Reply:
Here you get the message via `handle_info`, which doesn't even have the `from` argument. The way is to take the original `from` from the original `handle_call`, include it in the `:retransmit` message and use here.